python/cudf/cudf/benchmarks/get_datasets.py

```diff
@@ -18,6 +21,49 @@
 }
 
 
+def create_random_data(dtype, file_type, only_file):
+    file_dir = "cudf/benchmarks/cuio_data/"
+    file_path = os.path.join(
+        file_dir, "file_data" + str(dtype) + "." + file_type
+    )
+
+    if only_file:
+        return None, file_path, None
+
+    if dtype == "datetime64[s]":
+        n_samples = 2 ** 19
+    else:
+        n_samples = 2 ** 21
+    n_features = 2 ** 6
+    random_state = 23
+    np.random.seed(random_state)
+    X = np.random.rand(n_samples, n_features)
+
+    return X, file_path, n_features
+
+
+def create_cudf_dataset(dtype, file_type, only_file):
```

Comment:
We can merge `create_cudf_dataset`  with `create_pandas_dataset` and make `create_dataset` which returns pandas DF. Then we can absorb into cudf.DataFrame

Reply:
Don't we already have utilities that can be used here? AFAICT the fuzz testing data generator can be used. @galipremsagar can you comment on this?

Reply:
Mostly this https://github.com/rapidsai/cudf/blob/89938fafe3bfb32d029741d1d006896583b68579/python/cudf/cudf/tests/dataset_generator.py#L267

Reply:
+1 to what Ram mentioned. It basically returns a pyarrow table which can be converted to pandas dataframe using `cudf._fuzz_testing.utils.pyarrow_to_pandas`.